app/src/main/java/pm/gnosis/heimdall/di/modules/InterceptorsModule.kt

```diff
@@ -37,4 +38,15 @@ class InterceptorsModule {
             it.proceed(request.newBuilder().url(url).build())
         }
     }
+
+    @Provides
+    @Singleton
+    @IntoMap
+    @IntKey(1)
+    @Named(INTERCEPTORS_WITH_PRIORITY)
+    fun providesLoggingInterceptor(): Interceptor {
```

Comment:
Alternatively we could make the returned Interceptor nullable and use `mapNotNull` to avoid adding it if not in debug mode